scout/server/blueprints/dashboard/templates/dashboard/dashboard_general.html

```diff
@@ -81,8 +81,12 @@
       </div>
     </div>
     {% if pane in (1,2) %}
+    <br>
     <div class="bottom-align-text">
-      <br>Examples: case_id=<strong>18201</strong>, HPO-term=<strong>HP:0001166</strong>, synopsis=<strong>synopsis:epilepsy</strong>, panel=<strong>panel:NMD</strong>, phenotype group=<strong>PG:0100022</strong>, cohort=<strong>cohort:pedhep</strong>.
+      Query examples:
+      {% for elem in ["case:18201", "exact_pheno:HP:0001166", "synopsis:epilepsy", "panel:NMD", "pheno_group:HP:0001250", "status:active", "cohort:pedhep", "similar_case:1820", "similar_pheno:HP:0001166,HP:0001250", "pinned:POT1", "causative:POT1", "user:Kent"]%}
```

Comment:
Had to modify the instructions of the query here, because I've changed the prefix. Ideally this should be substituted by the same flask-wtf used in cases filter. But can be done in another PR